artichoke-backend/src/extn/core/symbol/mod.rs

```diff
@@ -54,6 +54,19 @@ impl Symbol {
             &[]
         }
     }
+
+    #[must_use]
+    pub fn inspect(self, interp: &'_ mut Artichoke) -> Vec<u8> {
+        if let Ok(Some(bytes)) = interp.lookup_symbol(self.into()) {
+            // pattern length + ':'
+            let mut inspect = Vec::with_capacity(bytes.len() + 1);
+            inspect.push(b':');
```

Comment:
To be correct, this function must parse the byte contents to determine if they should be wrapped in quotes or otherwise escaped. For example:

```
:""
: "$-www"
: "\xFF"
:"0123456789"
```

Reply:
I had trouble formatting this on mobile. Please forgive the extra spaces.